<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RocketCaveAdvancedWidget</class>
 <widget class="QWidget" name="RocketCaveAdvancedWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advanced Rocket Cave Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#RocketCaveAdvancedWidget {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(232, 234, 233, 255), stop:1 rgba(245, 246, 247, 255));
}

QWidget {
	font-family: &quot;Arial&quot;;
	font-size: 13px;
}

QLabel#presetsText, #editPresetText, #eyeSettings {
	font-size: 14px;
	font-weight: bold;
}

QLabel#noPresetsText{
	font-size: 14px;
	font-weight: bold;
	color: rgb(255, 0, 0);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="eyeSettings">
     <property name="styleSheet">
      <string notr="true">QLabel#eyeSettings{
	font-size: 14px;
}</string>
     </property>
     <property name="text">
      <string>Eye settings</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Eye position:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="eyeX">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-1000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="eyeY">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-1000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="eyeZ">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-1000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="updateEyePositionButton">
       <property name="text">
        <string>Update eye position</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="customMatrix">
     <property name="text">
      <string>Use custom views</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="editView1">
       <property name="text">
        <string>Edit view 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editView2">
       <property name="text">
        <string>Edit view 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editView3">
       <property name="text">
        <string>Edit view 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editView4">
       <property name="text">
        <string>Edit view 4</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
